systest/cdc/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - type: bind
         source: ./cdc_logs
         target: /cdc
-    command: /gobin/dgraph alpha --my=alpha1:7080 --zero=zero1:5080 --logtostderr
+    command: dgraph alpha --my=alpha1:7080 --zero=zero1:5080 --logtostderr
```

Comment:
Can the above /gobin bind point be removed?

Reply:
fixed, thanks